backend/.env.prod

```
@@ -0,0 +1,106 @@
+########################################################
+# Provide Your Own Values for the following variables.
+# For local development, you can reuse most of the values from the .env.example file, except for the following:
+# ########################################################
+# SERVER_OPENAI_API_KEY=<your_openai_api_key>
+# CLI_OPENAI_API_KEY=<your_openai_api_key_can_be_the_same_as_server_openai_api_key>
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)